README.rst

```diff
@@ -3,10 +3,22 @@ Sentry Patrol
 
 This is still in alpha state and in active development
 
+
+Setup
+~~~~~
+
+.. code:: bash
+
+    pipenv install --dev
+    python setup.py develop
+    export SENTRY_API_TOKEN=<your-token>
+
+
 Example
 ~~~~~~~
 
 This will list all events of a specific project
 
 .. code:: bash
-python -m patrol events <organization> <project>
+
+    patrol events <organization> <project>
```

Comment:
patrol already has two commands now :)
please add `patrol issues <organization> <project>`

Reply:
What about `patrol --help` instead of the list of all commands ?

Reply:
sounds better